include/mesh.h

```diff
@@ -73,6 +85,9 @@ class Mesh {
   //! Map to store element id and vertices coordinates
   std::map<unsigned, Eigen::VectorXd> elementcoordinates_;
 
+  //! Map to store element id and volume of the element
+  std::map<unsigned, double> volumes_;
+
```

Comment:
@kks32 didn't we decide not to store volumes?

Reply:
@cw646, I was just thinking it is cleaner and more consistent this way.

Previously you just pass the whole map to `io` and now that we are doing it in `mesh` I can make it not to store and to have just one function to `compute` and to `write`. What do you think?

Reply:
Material points will know it's volume, can we just iterate over points and print volumes?